tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -871,3 +871,139 @@ export const pod_hair_emissive: Feature<boolean> = {
   description: 'Want your flowers to glow in the dark?',
   component: CheckboxInput,
 };
+
+// SKYRAT EDIT ADDITION BEGIN - DEATH CONSEQUENCES QUIRK
+// all will be prefixed with DC to keep them all in one place
+export const dc_expand_options: Feature<boolean> = {
+  name: 'DC Options',
```

Comment:
```suggestion
  name: 'DC Config',
```
Since they get sorted alphabetically this will conveniently keep the checkbox above the rest of the options, which I think is preferable.